src/sentry/release_health/release_monitor/metrics.py

```diff
@@ -162,10 +162,10 @@ def fetch_project_release_health_totals(
                         env_name = indexer.reverse_resolve(row[env_key])
                         release_name = indexer.reverse_resolve(row[release_key])
                         row_totals = totals[row["project_id"]].setdefault(
-                            env_name, {"total_sessions": 0, "releases": defaultdict(int)}
+                            env_name, {"total_sessions": 0, "releases": defaultdict(int)}  # type: ignore
```

Comment:
This code's typing is wrong, but I don't know how to fix it. Doing something like adding an assertion `assert env_name is not None` actually just breaks tests.